prometheus-exporters-formula/prometheus-exporters/map.jinja

```diff
@@ -0,0 +1,11 @@
+{% set exporters = salt['grains.filter_by']({
+    'RedHat': {
+        'node_exporter_package': 'golang-github-prometheus-node_exporter',
+        'node_exporter_service': 'prometheus-node_exporter',
+    },
+    'Suse': {
+        'node_exporter_package': 'golang-github-prometheus-node_exporter',
+        'node_exporter_service': 'prometheus-node_exporter',
+    },
```

Comment:
No Ubuntu?

Reply:
Since this is just an MVP, not sure if we need to enable Ubuntu right away.

Reply:
We can add `Ubuntu`, it's easy, we just need to check what is the name of the package and service. Also `RedHat` at the moment is just a copy of the `Suse` names, will check if package or service name actually differs.

Reply:
Added support for Ubuntu (untested), the package should be available from Universe. Dropping support for RedHat for now as I could not easily find the package and service names.